docs/visualization/perses.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Perses support for Prometheus
+nav_title: Perses
+sort_rank: 3
+---
+
+[Perses](https://perses.dev) supports Prometheus since the beginning of the project.
```

Comment:
Maybe start with a short description of what Perses is?

Reply:
I think having a small description for Grafana would be nice too. But we can do that later.

Reply:
I can do that on another PR 👍

Reply:
done in https://github.com/prometheus/docs/pull/2674